apps/frontend/src/components/profile/Profile.tsx

```
@@ -0,0 +1,63 @@
+
+  return (
+    // TODO:: Can have a better way to adjust the layout(left margin) using flex
+    <div className="ml-[160px] w-1/2  bg-background p-16 text-foreground">
+      <UserInfo user={user} />
+      <Integrations user={user} />
+      <footer className="text-muted-foreground mb-28 mt-24 text-[16px]">
+        <p className="mb-2 font-semibold">march 0.1</p>
+        <p className="text-xs text-secondary-foreground">
+          — crafted for the makers to{" "}
+          <span className="text-primary-foreground">get things done</span>;
+        </p>
```

Biome

[error] There is a suspicious semicolon in the JSX element.

This is usually the result of a typo or some refactor gone wrong.
Remove the semicolon, or move it inside a JSX element.

(lint/suspicious/noSuspiciousSemicolonInJsx)